src/rest.ts

```diff
@@ -23,16 +28,16 @@ const CR = '\r'.charCodeAt(0); // always 13, but avoid magic constant
 
 const PODMAN_SYSTEM_ADDRESS = "/run/podman/podman.sock";
 
-/* uid: null for logged in session user, otherwise standard Unix user ID
- * Return { path, superuser } */
-function getAddress(uid) {
+export type Uid = number | null; // standard Unix UID or null for logged in session user
+
+function getAddress(uid: Uid): { path: string, superuser?: "require" | "try" } {
     if (uid === null) {
         // FIXME: make this async and call cockpit.user()
         const xrd = sessionStorage.getItem('XDG_RUNTIME_DIR');
         if (xrd)
-            return { path: xrd + "/podman/podman.sock", superuser: null };
+            return { path: xrd + "/podman/podman.sock" };
         console.warn("$XDG_RUNTIME_DIR is not present. Cannot use user service.");
-        return { path: "", superuser: null };
+        return { path: "" };
```

Comment:
This added line is not executed by any test.  [Details](https://logs-cockpit.apps.ocp.cloud.ci.centos.org/pull-2214-e5ac0ddf-20250801-075515-fedora-42-devel/Coverage/lcov/github-pr.diff.gcov.html)